scripts/create-symlinks.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+const { existsSync } = require( 'fs' );
+const mkdirp = require( 'mkdirp' );
+const { sync: symlinkOrCopy } = require( 'symlink-or-copy' );
+
+process.stdout.write( '\nCreating symlinks…\n' );
+
+mkdirp( 'node_modules/@wordpress', () => {
+	const symlinksToCreate = [
+		[ 'packages/browserslist-config', 'node_modules/@wordpress/browserslist-config' ],
+		[ 'packages/babel-preset-default', 'node_modules/@wordpress/babel-preset-default' ],
+		[ 'packages/jest-preset-default', 'node_modules/@wordpress/jest-preset-default' ],
+		[ 'packages/scripts', 'node_modules/@wordpress/scripts' ],
+		[ 'node_modules/@wordpress/scripts/bin/wp-scripts.js', 'node_modules/.bin/wp-scripts' ],
+	];
+	symlinksToCreate.forEach( ( [ srcPath, destPath ] ) => {
```

Comment:
I guess there's no async version of this module? Otherwise seems like it'd be simple / advisable to write as a `Promise.all`

Reply:
From README of this npm package:

> There intentionally isn't an asynchronoukks version. It's not clear that we need or want one. Before sending a patch to add an async version, please share your use case on the issue tracker.

They provide only sync version.